www/docs/en/dev/guide/platforms/ios/index.md

```diff
@@ -74,190 +67,123 @@ There are two ways to download Xcode:
 
-## Deploy the app
-
-To deploy the app on a connected iOS device:
-
-        $ cordova run ios --device
-To deploy the app on a default iOS emulator:
-
-        $ cordova emulate ios
-
-You can use __cordova run ios --list__ to see all available targets and 
-__cordova run ios --target=target_name__ to run application on a specific 
-device or emulator (for example,  `cordova run ios --target="iPhone-6"`).
-
-You can also use __cordova run --help__ to see additional build and run
-options.
-
-## Open a Project in the SDK
-
-Once ios platform is added to your project, you can open it from 
-within Xcode. Double-click to open the `hello/platforms/ios/hello.xcodeproj`
-file. The screen should look like this:
-
-![]({{ site.baseurl }}/static/img/guide/platforms/ios/helloworld_project.png)
+## Signing an App
 
-## Deploy to Emulator
+First, you should read through the [Code Signing Support Page](https://developer.apple.com/support/code-signing/) 
+and the [App Distribution Workflows](https://developer.apple.com/library/prerelease/ios/documentation/IDEs/Conceptual/AppDistributionGuide/Introduction/Introduction.html).
 
-To preview the app in the iOS emulator:
+### Using Flags
 
-1. Make sure the _.xcodeproj_ file is selected in the left panel.
+To sign an app, you need the following parameters:
 
-2. Select the __hello__ app in the panel immediately to the right.
+| Parameter                | Flag                     | Description
+|--------------------------|--------------------------|-----------------------------------
+| Code Sign Identity       | `--codeSignIdentity`     | Code signing identity to use for signing. It can be created with Xcode and added to your keychain.
+| Provisioning Profile     | `--provisioningProfile`  | GUID of the provisioning profile to be used for signing. It is copied here on your Mac: ~/Library/MobileDevice/Provisioning\ Profiles/. Opening it in a text editor, you can find the GUID which needs to be specified here.
+| Code Sign Resource Rules | `--codesignResourceRules`| (Optional) Used to control which files in a bundle should be sealed by a code signature. For more details, read [The OS X Code Signing In Depth article](https://developer.apple.com/library/mac/technotes/tn2206/_index.html#//apple_ref/doc/uid/DTS40007919-CH1-TNTAG206) 
 
-3. Select the intended device from the toolbar's __Scheme__ menu, such
-   as the iPhone 6.0 Simulator as highlighted here:
+### Using build.json
 
-   ![]({{ site.baseurl }}/static/img/guide/platforms/ios/select_xcode_scheme.png)
+Alternatively, you could specify them in a build configuration file (`build.json`)
+using the `--buildConfig` argument to the same commands. Here's a sample of a
+build configuration file:
 
-4. Press the __Run__ button that appears in the same toolbar to the
-   left of the __Scheme__. That builds, deploys and runs the
-   application in the emulator. A separate emulator application opens
-   to display the app:
+    {
```

Comment:
this JSON is not well formated, it should be `{
	"ios": {
		"debug": {
			"codeSignIdentitiy": "iPhone Development",
			"provisioningProfile": "926c2bd6-8de9-4c2f-8407-1016d2d12954"
		},
		"release": {
			"codeSignIdentitiy": "iPhone Distribution",
			"provisioningProfile": "70f699ad-faf1-4adE-8fea-9d84738fb306"
		}
	}
}`

See https://issues.apache.org/jira/browse/CB-9730